packages/client/lib/CredentialOffer/CredentialOfferClient.ts

```diff
@@ -0,0 +1,42 @@
+import { CredentialOfferWithBaseURL, OpenId4VCIVersion } from '@sphereon/openid4vci-common';
+import Debug from 'debug';
+
+import { CredentialIssuanceClient } from './CredentialIssuanceClient';
+
+const debug = Debug('sphereon:openid4vci:credentialOffer');
+
+// FIXME implement the function
+export class CredentialOfferClient implements CredentialIssuanceClient {
```

Comment:
Why is a new client necessary? The change is a simple change in naming/URI. The more classes we introduce the more maintenance and the more chances for bugs.

Reply:
reverted.